semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -89,17 +89,17 @@ trait TextDocumentOps { self: SemanticdbOps =>
             mstarts(mstart1) = mname
             mends(mend1) = mname
           }
-          private def indexArgNames(mapp: m.Tree, mnames: List[m.Name]): Unit = {
-            if (mnames.isEmpty) return
-            todo ++= mnames
-            val mstart1 = mapp.tokens
-              .dropWhile(_.is[m.Token.LeftParen])
-              .headOption
-              .map(_.start)
-              .getOrElse(-1)
-            // only add names for the top-level term.apply of a curried function application.
-            if (!margnames.contains(mstart1))
-              margnames(mstart1) = mnames
+          private def indexArgNames(mapp: m.Tree): Unit = {
```

Comment:
isn't this a reworked `loop` function from below? You can remove it below if it's not needed.

Reply:
Removed.